self-api/src/main/java/com/selfxdsd/api/ApiToken.java

```diff
@@ -0,0 +1,29 @@
+package com.selfxdsd.api;
+
+import java.time.LocalDateTime;
+
+/**
+ * API Token.
```

Comment:
@andreoss Each class JavaDoc should have the following tags:

```
@author Name (E-Mail)
@version $Id$
@since <<next release version>>
```

In this case, it should be ``@since 0.0.61``.
Author is optional, write it only if you want to.

Reply:
@amihaiemil Done.